openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -29,6 +29,21 @@ def get_current_site_configuration():
         return None
 
 
+def get_current_site_configuration_values(default=None):
+    """
+    Returns `SiteConfiguration.site_values` for current site.
```

Comment:
Arg, one last thing I missed, sorry. Docstring should specify args, etc. Maybe something like this? Please edit to make better, I'm sleepy at the moment.

```
    """
    Return SiteConfiguration.site_values for the current site.
    Args:
        default (dict): default value to return if key is not found in the configuration
    Returns:
        (dict) Configuration value for the current site or `{}` if default is not specified 
    """
```

Reply:
@sarina, sorry for coming back to this review comment late. I've improved the docstring as suggested.